conf/modules.config

```diff
@@ -181,10 +181,12 @@ if (!(params.skip_fastqc || params.skip_qc)) {
 if (!params.skip_trimming) {
     process {
         withName: '.*:FASTQC_UMITOOLS_TRIMGALORE:TRIMGALORE' {
-            ext.args   = [
-                '--fastqc',
-                params.trim_nextseq > 0 ? "--nextseq ${params.trim_nextseq}" : ''
-            ].join(' ').trim()
+            ext.args   = {
+                [
+                    "--fastqc_args '-t ${task.cpus}' ",
```

Comment:
``--fastqc_args`` automatically triggers fastqc but with the additional argument of passing the number of threads to the fastqc wrapper.

Reply:
Nice!